docs/asciidoc/faq.adoc

```diff
@@ -85,3 +86,18 @@ Format
 Reactor Netty adds information for the connection at the beginning of every log record (when this is possible).
 Use the id of the connection in order to extract all log records for a particular HTTP request.
 For more information see <<faq.logging-prefix>>
+
+[[faq.memory-leaks]]
+== How can I debug a memory leak?
+By default, Reactor Netty uses a direct memory as this is more performant
+when there are many native I/O operations (working with sockets) as this can remove the copying operations.
+As allocation and deallocation are expensive operations, the buffers, used in Reactor Netty by default, are pooled objects.
```

Comment:
Change the commas around "used in Reactor Netty by default" to parentheses. While technically correct, with other commas nearby, it is hard to parse. Replacing those two commas with parentheses will clarify the sentence.